test/built-ins/TypedArrays/called-with-length-return-object.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+id: sec-typedarray-length
+description: >
+  Return a TypedArray object
+info: >
+  22.2.4.2 TypedArray ( length )
+
+  This description applies only if the TypedArray function is called with at
+  least one argument and the Type of the first argument is not Object.
+
+  ...
+  8. Return ? AllocateTypedArray(constructorName, NewTarget,
+  %TypedArrayPrototype%, elementLength).
+
+  22.2.4.2.1 Runtime Semantics: AllocateTypedArray (constructorName, newTarget,
+  defaultProto [ , length ])
+
+  ...
+  7. Return obj
+includes: [testTypedArray.js]
+---*/
+
+testWithTypedArrayConstructors(function(TA) {
+  var typedArray = new TA(4);
+  var bytesPerElement = TA.BYTES_PER_ELEMENT;
+  var length = typedArray.length;
+
+  assert.sameValue(length, 4, "length");
+  assert.sameValue(
```

Comment:
For consistency with the other tests, I'd include an assertion for the value of the instance's `constructor` property